compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolEnv.java

```diff
@@ -263,10 +263,6 @@ public static SymbolEnv createStreamingQueryEnv(BLangStreamingQueryStatement nod
         return createEnv(node, env);
     }
 
-    public static SymbolEnv createStreamingInputEnv(BLangNode node, SymbolEnv env) {
-        return createEnv(node, env);
-    }
-
```

Comment:
Can we remove public methods since this goes to patch branch?

Reply:
+1 yeah.. should revert that..